doc/source/data/examples/index.rst

```diff
@@ -4,6 +4,23 @@
 Ray Data Examples
 =================
 
+..
+   Include all examples in a hidden toctree so Sphinx build does not complain.
+
+.. toctree::
+    :hidden:
+
+    huggingface_vit_batch_prediction
+    pytorch_resnet_batch_prediction
+    batch_inference_object_detection
+    nyc_taxi_basic_processing
+    batch_training
+    ocr_example
+    random-access
+    custom-datasource
```

Comment:
These will show up in the navbar.

Reply:
yep, as discussed offline, doesn't seem there's an easy way to avoid this